Cargo.toml

```diff
@@ -26,9 +26,9 @@ libc = "0.2"
 parking_lot = "0.12.2"
 paste = "1.0.14"
 raw-cpuid = "11.0.1"
-strum = { version = "0.26", features = ["derive"] }
+strum = { version = "0.27", features = ["derive"] }
 to_method = "1.1.0"
-zerocopy = { version = "0.7.32", features = ["derive"] }
+zerocopy = { version = "0.8", features = ["derive"] }
```

Comment:
I think `zerocopy` 0.8 had a bunch of breaking changes that we need to fix to update to this.